doc/api.md

```diff
@@ -237,6 +241,12 @@ Note that, when information coming from devices, this means measures, are not de
 device, the IoT agent will store that information into the destination entity using the same attribute name than the
 measure name, unless `explicitAttrs` is defined. Measures `id` or `type` names are invalid, and will be ignored.
 
+## Device autoprovision and entity creation
```

Comment:
Is this section included in ToC?

Reply:
Added in 557f75aa143a9430a92ef44859a61bb8f6c22797